src/displayapp/screens/settings/SettingWatchFace.cpp

```diff
@@ -26,7 +26,10 @@ SettingWatchFace::SettingWatchFace(Pinetime::Applications::DisplayApp* app,
   : app {app},
     settingsController {settingsController},
     filesystem {filesystem},
-    screens {app, 0, CreateScreenList(), Screens::ScreenListModes::UpDown} {
+    screens {app,
+             static_cast<uint8_t>(static_cast<uint8_t>(settingsController.GetWatchFace()) / 4),
```

Comment:
I think it would make more sense to cast the constant rather than the result

Reply:
After a lot of testing, casting the constant results in narrowing compiler errors. The above is the only casting that the compiler would accept (not even a warning thrown).

Reply:
You're totally right. All divisions get promoted to at least `int` size apparently. So I think we can leave this as is. Sorry!